packages/dashboard/src/features/compute/hooks/useComputeCapabilities.ts

```
@@ -0,0 +1,41 @@
+  capabilities: ComputeCapabilitiesSchema | undefined;
+  provider: string | undefined;
+  isLoading: boolean;
+} {
+  const { metadata, isLoading } = useMetadata();
+  const compute = metadata?.compute;
+  const name = provider ?? compute?.defaultProvider;
+  // A provider name off a service row is a plain string, and one the backend no
+  // longer reports should simply read as unknown rather than fail to compile.
+  const providers = compute?.providers as
+    | Record<string, ComputeCapabilitiesSchema | undefined>
+    | undefined;
```

ESLint

[error] Replace `|·Record<string,·ComputeCapabilitiesSchema·|·undefined>⏎···` with `Record<string,·ComputeCapabilitiesSchema·|·undefined>`

(prettier/prettier)